tsc failed to build with `import { type xxx} `

`import { type ` syntax seems not working for tsc, even this project can be built successfully, however if we try to import this project from another tsc complains as follows:
```
../../common/temp/node_modules/.pnpm/@ampproject+remapping@2.1.0/node_modules/@ampproject/remapping/dist/types/source-map.d.ts:1:15 - error TS1005: ',' expected.

1 import { type TraceMap } from '@jridgewell/trace-mapping';
```

--- a/src/source-map.ts
+++ b/src/source-map.ts
@@ -1,4 +1,5 @@
-import { type TraceMap, encodedMappings, decodedMappings } from '@jridgewell/trace-mapping';
+import type { TraceMap } from '@jridgewell/trace-mapping';
+import { encodedMappings, decodedMappings } from '@jridgewell/trace-mapping';
 import type { DecodedSourceMap, RawSourceMap, Options } from './types';
 
 /**

